packages/tools/create-package/src/deploy-package.ts

```diff
@@ -0,0 +1,47 @@
+import fs from 'node:fs/promises';
+import path from 'node:path';
+import type { Project } from './types.js';
+
+const DIRNAME = path.dirname(new URL(import.meta.url).pathname);
+const ROOT_DIR = path.join(DIRNAME, '../../../..');
+const TEMPLATE_DIR = path.join(ROOT_DIR, 'packages/templates/package');
+const RUSH_CONFIG_PATH = path.join(ROOT_DIR, 'rush.json');
+
+export const deployPackage = async ({
+  dir,
+  name,
+  description,
+  repoUrl,
+  type,
+  shouldPublish,
+}: Project) => {
+  const targetDir = path.join(ROOT_DIR, dir);
+  const manifestFilePath = path.join(targetDir, 'package.json');
+
+  await fs.cp(TEMPLATE_DIR, targetDir, {
+    recursive: true,
+    filter: (src) => !/(.rush|node_modules|log|temp\/)/.test(src),
+  });
```

Comment:
- For this particular tool/purpose performance doesn't _really_ matter.
- I definitely would prefer to start/educate/use async when possible. Why do you think it is a preference? It's not from a Node.js perspective.

Let's use `...Sync` methods only when we have to, it has no real benefits otherwise. And if performance is actually an issue we better back the claim up with numbers.